templates/gitlab-runner.yaml

```diff
@@ -92,9 +92,10 @@ Resources:
       Path: /
       ManagedPolicyArns:
         - !ImportValue
-          'Fn::Sub': '${AWS::Region}-essentials-ManagedInstanceRoleArn'
+          'Fn::Sub': '${AWS::Region}-essentials-TagRootVolumePolicy'
```

Comment:
didn't we want to replace `TagRootVolumePolicy`  with `ManagedInstanceRoleArn`?  why are you switching back?

Reply:
I had to change this a bit.. We're exporting an instance role from essentials, but I need to include policies in this list. I've included the `AmazonSSMManagedInstanceCore` on line 98, which when combined with `TagRootVolumePolicy` makes up what we export from essentials.